tools/myst/host/exec_linux.c

```diff
@@ -82,6 +82,27 @@ Options:\n\
                             it encounters an unimplemented syscall\n\
                             'true' implies the syscall would not terminate\n\
                             and instead return ENOSYS.\n\
+    --strace\n\
+                         -- Use this option to display the strace of the execution\n\
+    --strace-failing\n\
+                         -- When specified, all syscalls that fail will be logged.\n\
+                           Other syscalls will not be logged, unless specified via \n\
+                           filter (see below).\n\
+                           Set a breakpoing in _strace_failure_hook to stop \n\
```

Comment:
Is this a debugging suggestion?

Reply:
Yes I think so, picked it up from the PR that added this feature

Reply:
We can also suggest using syscall number as a condition in the breakpoint.
Also there is a typo in breakpoing.

Reply:
Added that wording